ui/components/ui/account-list/index.scss

```diff
@@ -0,0 +1,140 @@
+.choose-account-list {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+
+  &__header--one-item,
+  &__header--two-items {
+    display: flex;
+    flex: 0;
+    margin-top: 36px;
+    width: 100%;
+    padding-left: 15px;
```

Comment:
Let's use `padding-inline-start` and `padding-inline-end` for better RTL support.

Reply:
We can use `margin-inline-start` and `margin-inline-end`  elsewhere in this file.

Reply:
Pushed up some changes, please let me know if that works.